@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -578,11 +584,11 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
             const computeTexCoords =
                 texCoordType === TextureCoordinateType.TileSpace
                     ? (pos: THREE.Vector3) => {
-                          const { x: u, y: v } = projection
-                              .reprojectPoint(webMercatorProjection, pos, texCoord)
-                              .sub(tileBounds.min)
-                              .divide(tileSize);
-                          return { u, v };
+                          const { x: u, y: v } = pos
+                              .clone()
+                              .sub(tileDataBounds.min)
+                              .divide(tileDataSize);
+                          return { u, v: 1 - v };
```

Comment:
Computes the UVs in the OMV tile space (Y points down). The previous code was computing the UVs in the target projection space (webmercator with Y up or spherical globe). Projecting the UVs in spherical globe doesn't really work with the subdivision modifier, so it's better to keep these attributes in their original space.